src/core/containers/HandleLogin/index.js

```diff
@@ -33,7 +34,8 @@ class HandleLogin extends React.Component {
   }
 
   render() {
-    const { code, state } = this.props.location.query;
+    const { location } = this.props;
+    const { code, state } = location.query;
     const { error } = this.state;
```

Comment:
I don't think we need to store this in redux. I'll add a log statement to the `catch` that sets it.